samples/Foundatio.ReceiverConsole/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
```

Comment:
remove commented out lines?

Reply:
We should be able to drop all assembly info's as well or at least just provide a title and description as we do in the Foundatio Project (logging branch)